sql-statements/sql-statement-admin-check-table-index.md

```diff
@@ -6,7 +6,24 @@ category: reference
 
 # ADMIN CHECK [TABLE|INDEX]
 
-The `ADMIN CHECK [TABLE|INDEX]` statement checks for data consistency of tables and indexes. It does not support checking [FOREIGN KEY constraints](/foreign-key.md).
+The `ADMIN CHECK [TABLE|INDEX]` statement checks for data consistency of tables and indexes. 
+
+It does not support:
+
+- checking [FOREIGN KEY constraints](/foreign-key.md).
+- checking the PRIMARY KEY index if a [clustered primary key](/clustered-indexes.md) is used.
+
+If `ADMIN CHECK [TABLE|INDEX]` finds any issues these can be fixed by recreating the index by dropping the index and adding it again. 
```

Comment:
Or `admin recover` or `admin cleanup`.
And it is related to `troubleshoot-data-inconsistency-errors.md`. We can add a link here.